### ISSUE TITLE
Add docker for sillytavern-extra to ghcr.io 

### ISSUE THREAD

**Issue by rhamblen** [SillyTavern/SillyTavern-Extras#225]
can we add the docker file to ghcr.io

I use unraid and its much harder add and build a docker file from scratch.  Especially when the files are updated.
If you just add it to ghcr.io the process is much easier.

thanks

**Comment by Webifi** [comment]
Yes, this would be helpful.

**Comment by zachferr** [comment]
+1 to this, but also for anyone having issues building the Dockerfile (at least with Apple Silicon), I was able to get it to build by changing line 1 to:
`FROM --platform=linux/amd64 nvidia/cuda:11.7.1-cudnn8-runtime-ubuntu20.04` (I may end up changing the image regular ubuntu instead of the nvidia cuda build since idt I need it)

and lines 9-10 to
```
RUN apt-get update && apt-get install -y --no-install-recommends \
        python3 python3-venv wget build-essential libgtk-3-dev
```
and make sure you're using the correct requirements.txt.

lastly if youre on apple silicon you need to remove the nvidia related sections from the docker compose

Just a heads up it takes a long time to build. Hope this helps! (and I'm still having trouble connecting to the server and getting it working but at least past step 1 haha)

**Comment by mrguymiah** [comment]
Agreed. I'm currently building it manually to try and figure it all out and getting it working. This project could definitely use some love in regards to its docker implementation.
